README.md

```diff
@@ -1 +1,5 @@
-# review-project
\ No newline at end of file
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
```

Comment:
Readme file should have more content sections to describe your project, why don't you add the list of technologies used and how to run them? It would make it more professional